src/pages/about.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { SharedLayout } from '../components/sharedLayout';
+
+export function getStaticProps() {
+  return {
+    props: { pkgVersion: require('@lifeomic/chromicons/package.json').version },
+  };
+}
+
+const About = ({ pkgVersion }) => {
+  return <SharedLayout pkgVersion={pkgVersion}>About</SharedLayout>;
+};
+
+export default About;
```

Comment:
If this component doesn't have any content yet, should we wait to add it to the app until we have the page built out?